map/bookmark_helpers.cpp

```diff
@@ -370,46 +371,49 @@ std::string GetKMLorGPXPath(std::string const & filePath)
     if (kmlData == nullptr)
       return {};
 
-    fileSavePath = GenerateValidAndUniqueFilePathForKML(base::FileNameFromFullPath(filePath));
+    auto fileSavePath = GenerateValidAndUniqueFilePathForKML(base::FileNameFromFullPath(filePath));
     if (!SaveKmlFileByExt(*kmlData, fileSavePath))
       return {};
+    return {std::move(fileSavePath)};
   }
   else if (fileExt == kKmzExtension)
   {
-    // Extract KML file from KMZ archive and save to temp KML with unique name.
+    // Extract KML files from KMZ archive and save to temp KMLs with unique name.
+    std::vector<std::string> kmlFilePaths;
     try
     {
       ZipFileReader::FileList files;
       ZipFileReader::FilesList(filePath, files);
-      std::string kmlFileName;
-      std::string ext;
-      for (size_t i = 0; i < files.size(); ++i)
+      std::string indexFileName;
+      std::vector<std::string> kmlFileNames;
```

Comment:
It's better to declare variables as close to the place of their usage as possible. Then it's easier to read the code, and in some cases of early exit, the code works faster.

Reply:
Both variables are not used now (not sure why it was not highlighted in clion) - removed both of them